site/src/components/components/LivePreview.module.css

```diff
@@ -25,6 +25,11 @@
   align-items: center;
   justify-content: center;
   gap: calc(2 * var(--salt-size-unit));
+  flex-wrap: wrap;
+}
+
+.darkMode {
+  background-color: var(--salt-color-gray-900);
```

Comment:
This looks pretty fragile. Is there a way to have the Provider wrap this and set the background?